### ISSUE TITLE
Troubleshooting Appropedia Recipe

### ISSUE THREAD

**Issue by RavanJAltaie** [openzim/mwoffliner#1959]
As per this https://github.com/openzim/zim-requests/issues/763, please check why this recipe is failing while we have already a successful resulted file in the past.

The link to the recipe on the Zimfarm:
https://farm.openzim.org/recipes/appropedia_en_all

The link to last working ZIM (if any):
https://download.kiwix.org/zim/other/appropedia_en_all_maxi_2021-03.zim

The error I get:
 Error downloading article Copper_T380A_IUD_Insertion
[error] Failed to run mwoffliner after [1461s]

**Comment by benoit74** [comment]
Just like in https://github.com/openzim/mwoffliner/issues/1956, we have (at least) one article which is failing.

**Comment by kelson42** [comment]
@benoit74 AFAIK this issue has been fixed and we don't suffer of it anymore. Good to close?

**Comment by benoit74** [comment]
Yep, nothing left to be solved at scraper level (recipe is failing because appropedia api.php is not reliable at all ... not our fault ...)
